docs(connector): add Aliyun DM README zh-CN (#1481)
logto-io/logto: packages/connector-aliyun-dm/README.md

  - [Set up an email service in Aliyun DirectMail Console](#set-up-an-email-service-in-aliyun-directmail-console)
    - [Register Aliyun account](#register-aliyun-account)
    - [Enable and configure Aliyun Direct Mail](#enable-and-configure-aliyun-direct-mail)
  - [Compose the connector JSON](#compose-the-connector-json)
    - [Test WeChat web connector](#test-wechat-web-connector)
    - [Config types](#config-types)
- [阿里云邮件连接器](#阿里云邮件连接器)
  - [开始上手](#开始上手)
  - [在阿里云邮件服务控制台中配置一个邮件服务](#在阿里云邮件服务控制台中配置一个邮件服务)
    - [注册阿里云帐号](#注册阿里云帐号)
    - [启用并配置阿里云邮件服务](#启用并配置阿里云邮件服务)
  - [编写连接器的 JSON](#编写连接器的-json)
    - [测试阿里云邮件连接器](#测试阿里云邮件连接器)
    - [配置类型](#配置类型)

## Get started

Aliyun is a primary cloud service provider in Asia, offering many cloud services, including DM (direct mail). Aliyun DM Connector is a plugin provided by the Logto team to call the Aliyun DM service APIs, with the help of which Logto end-users can register and sign in to their Logto account via mail verification code (or in other words, passcode).

## Set up an email service in Aliyun DirectMail Console

> 💡 **Tip**
> 
> You can skip some sections if you have already finished.

### Register Aliyun account

> Go to the [Aliyun website](https://cn.aliyun.com/) and register your Aliyun account if you don't have one.

Reviewer: ```suggestion
Head to [Aliyun](https://aliyun.com/) and create your Aliyun account if you don't have one.
```

# Aliyun direct mail connector

The official Logto connector for Aliyun connector for direct mail service.

> 阿里云邮件推送服务官方 Logto 连接器 [中文文档](#阿里云邮件连接器)

Reviewer: ```suggestion
阿里云邮件推送服务 Logto 官方连接器 [中文文档](#阿里云邮件连接器)
```

  - [Get started](#get-started)
  - [Set up an email service in Aliyun DirectMail Console](#set-up-an-email-service-in-aliyun-directmail-console)
    - [Register Aliyun account](#register-aliyun-account)
    - [Enable and configure Aliyun Direct Mail](#enable-and-configure-aliyun-direct-mail)
  - [Compose the connector JSON](#compose-the-connector-json)
    - [Test Aliyun DM connector](#test-aliyun-dm-connector)
    - [Config types](#config-types)
- [阿里云邮件连接器](#阿里云邮件连接器)
  - [开始上手](#开始上手)
  - [在阿里云邮件服务控制台中配置一个邮件服务](#在阿里云邮件服务控制台中配置一个邮件服务)
    - [注册阿里云帐号](#注册阿里云帐号)
    - [启用并配置阿里云邮件服务](#启用并配置阿里云邮件服务)
  - [编写连接器的 JSON](#编写连接器的-json)
    - [测试阿里云邮件连接器](#测试阿里云邮件连接器)
    - [配置类型](#配置类型)

## Get started

Aliyun is a primary cloud service provider in Asia, offering many cloud services, including DM (direct mail). Aliyun DM Connector is a plugin provided by the Logto team to call the Aliyun DM service APIs, with the help of which Logto end-users can register and sign in to their Logto account via mail verification code (or in other words, passcode).

## Set up an email service in Aliyun DirectMail Console

> 💡 **Tip**
> 
> You can skip some sections if you have already finished.

> ### Register Aliyun account

Reviewer: ```suggestion
### Create an Aliyun account
```

| subject             | string      | N/A                              |
| content             | string      | N/A                              |
| usageType           | enum string | 'Register' \| 'SignIn' \| 'Test' |

# 阿里云邮件连接器

## 开始上手

阿里云是亚洲地区一个重要的云服务厂商，提供了包括邮件服务在内的诸多云服务。

> 阿里云邮件连接器是 Logto 团队官方提供的，帮助终端用户能够通过邮件收取验证码从而能够注册和登录 Logto 帐号的插件工具。

Reviewer: ```suggestion
本连接器是 Logto 官方提供的阿里云邮件连接器，帮助终端用户通过邮件验证码进行登录注册。
```

## 在阿里云邮件服务控制台中配置一个邮件服务

> 💡 **Tip**
> 
> 你可以跳过已经完成的部分。

### 注册阿里云帐号

> 来到 [阿里云网站](https://cn.aliyun.com/) 并完成帐号的注册。

Reviewer: ```suggestion
前往 [阿里云](https://aliyun.com/) 并完成帐号的注册。
```

### 启用并配置阿里云邮件服务

> 来到 [阿里云邮件服务页面](https://www.aliyun.com/product/directmail) 然后登录。点击页面左上的「申请开通」按钮以开通邮件服务并开始配置流程。

Reviewer: ditto，前往

Reviewer: 点击 -> 点按

> 来到 [阿里云邮件服务页面](https://www.aliyun.com/product/directmail) 然后登录。点按页面左上的「申请开通」按钮以开通邮件服务并开始配置流程。
> 
> 从 [邮件服务管理控制台页面](https://dm.console.aliyun.com/) 开始：

Reviewer: the link itself implies it's a webpage. so no need for further explanation

```suggestion
来到 [阿里云邮件服务](https://www.aliyun.com/product/directmail) 然后登录。点按页面左上的「申请开通」按钮以开通邮件服务并开始配置流程。

从 [邮件服务管理控制台](https://dm.console.aliyun.com/) 开始：
```

1. 从侧边栏进入到「发信域名」，点按「新建域名」并完成指引。
2. 依次配置好「发信地址」和「邮件标签」。

在完成了设置之后，这里提供了两种测试的方法：

> - 前往 [邮件服务管理控制台概览页](https://dm.console.aliyun.com/)，在该页面底部找到「操作引导」框并点按「发送邮件」。你可以找到很多不同的测试方法。

Reviewer: ```suggestion
- 前往 [邮件服务管理控制台概览](https://dm.console.aliyun.com/)，在该页面底部找到「操作引导」框并点按「发送邮件」。你可以找到很多不同的测试方法。
```

> - 从侧边栏开始，选择「发送邮件」->「发送邮件」，在这里你可以「新建发送任务」来测试。

Reviewer: ```suggestion
- 在侧边栏中选择「发送邮件」->「发送邮件」，在这里你可以「新建发送任务」来测试。
```

will be great if we can have a screenshot here.

## 编写连接器的 JSON

> 1. 在 [邮件服务管理控制台页面](https://dm.console.aliyun.com/)，鼠标停在右上角你的头像上，进入「AccessKey 管理」，点按「创建 AccessKey」。完成了安全验证之后，你会得到一对「AccessKey ID」和「AccessKey Secret」，请妥善保管他们。
> 2. 从 [邮件服务管理控制台页面](https://dm.console.aliyun.com/) 的侧边栏，分别进入「发信地址」和「邮件标签」。这里你可以找到之前创建的 _发信地址_ 和 _邮件标签_。

Reviewer: ```suggestion
1. 在 [邮件服务管理控制台](https://dm.console.aliyun.com/)，鼠标停在右上角你的头像上，进入「AccessKey 管理」，点按「创建 AccessKey」。完成了安全验证之后，你会得到一对「AccessKey ID」和「AccessKey Secret」，请妥善保管他们。
2. 从 [邮件服务管理控制台](https://dm.console.aliyun.com/) 的侧边栏，分别进入「发信地址」和「邮件标签」。这里你可以找到之前创建的 _发信地址_ 和 _邮件标签_。
```

3. 完成阿里云邮件服务连接器的设置：
    - 用你在步骤 1 中拿到的一对「AccessKey ID」和「AccessKey Secret」来分别填入 `accessKeyId` 和 `accessKeySecret`。
    - 用步骤 2 中的 _发信地址_ 和 _邮件标签_ 填写 `accountName` 和 `fromAlias`。
    - 你可以添加多个邮件服务模板以应对不同的用户场景。这里展示填写单个模板的例子：
      - 在 `subject` 栏填写发送邮件的 _标题_。
      - 在 `content` 栏中填写字符形式的内容。不要忘了在内容中插入 `{{code}}` 占位符，在真实发送时他会被替换成随机生成的验证码。

>       - `usageType` 栏填写 `Register`，`SignIn` 或者 `Test` 其中之一以分别对应 _注册_，_登录_ 和 _测试_ 的不同场景。

Reviewer: will it be better to mention `usageType` is a Logto prop?

这是一个阿里云邮件服务连接器 JSON 配置的样例。

```json
{
    "accessKeyId": "<your-access-key-id>",
    "accessKeySecret": "<your-access-key-secret>",
    "accountName": "<noreply@logto.io>",

>     "fromAlias": "<logto>",

Reviewer: just double check: are these props all required?

Reviewer: move fwd if so
